lib/v2/oreno3d/characters.js

```diff
@@ -0,0 +1,71 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const cheerio = require('cheerio');
+const get_sec_page_data = require("./get_sec_page_data");
+
+const rootUrl = 'https://oreno3d.com';
+
+const sortRename = {
+    favorites: '高評価',
+    hot: '急上昇',
+    latest: '新着',
+    popularity: '人気',
+};
+
+const selector = 'a.box';
+module.exports = async (ctx) => {
+    // 下载
+    const sort = ctx.params.sort ?? 'latest';
+    const characterid = ctx.params.characterid ;
+    const userUrl = `${rootUrl}/characters/${characterid}?sort=${sort}`;
+    const response = await got.get(userUrl);
```

Comment:
```suggestion
    const response = await got(userUrl);
```
https://github.com/DIYgod/RSSHub/pull/10458#discussion_r945180004